test/Altinn.App.Api.Tests/Data/apps/tdd/task-action/config/texts/resource.nb.json

```diff
@@ -28,6 +28,10 @@
     {
       "id": "Side1.Button-Lj0cYG.title",
       "value": "Avvis"
+    },
+    {
+        "id": "PdfPreviewText",
+        "value": "Dette er en forhåndsvisning"
```

Comment:
This is an app we use for testing, not the default texts used everywhere. Also, the capitalisation is wrong. It should be `"pdfPreviewText"`

Reply:
RIght, then Im not sure how we set default texts?

Reply:
We don't have that functionality in the backend (yet).

Reply:
Actually we don't translate anything in the backend, so there was no need. Pdf footer kind of throws a wrench in that assumption.

Reply:
Right, guess I could submit the translated text in the request from FE?

Reply:
Though that would allow a user to set whatever text they want ofc. Like "Signed by Bjørn Dælie" 😆

Reply:
I think we should just set the text in bokmål for now right?

Reply:
No, just store the defaults (for all languages) inline in C# and let the app developer override through the use of `resource_{lang}.json`

A set of default translations can be implemented later if we need it.

Reply:
Note the race condition in comments here. You commented twice before my reply